packages/tangle/timemanager.go

```diff
@@ -107,17 +110,22 @@ func (t *TimeManager) LastConfirmedMessage() LastConfirmedMessage {
 	return t.lastConfirmedMessage
 }
 
-// Time returns the TangleTime, i.e., the issuing time of the last confirmed message.
-func (t *TimeManager) Time() time.Time {
+// CurrentTangleTime returns the CTT, i.e., the issuing time of the last confirmed message.
+func (t *TimeManager) CurrentTangleTime() time.Time {
```

Comment:
Shouldn't it be ConfirmedTangleTime instead as we have FinalizedTangleTime below?

Reply:
Done